hummingbot/connector/exchange/ascend_ex/ascend_ex_api_order_book_data_source.py

```diff
@@ -311,3 +426,8 @@ async def listen_for_order_book_snapshots(self, ev_loop: asyncio.BaseEventLoop,
             except Exception:
                 self.logger().error("Unexpected error.", exc_info=True)
                 await asyncio.sleep(5.0)
+
+    async def _get_ws_assistant(self) -> WSAssistant:
+        if self._ws_assistant is None:
+            self._ws_assistant = await self._api_factory.get_ws_assistant()
+        return self._ws_assistant
```

Comment:
Since this is used only in one place both the function and the variable can be removed

Reply:
I put it there for consistency with other classes. 
Should I remove it then?

Reply:
Removed